src/main/scala/nl.knaw.dans.easy.solr4files/package.scala

```diff
@@ -49,6 +49,10 @@ package object solr4files extends DebugEnhancedLogging {
   case class HttpStatusException(msg: String, response: HttpResponse[String])
     extends Exception(s"$msg - ${ response.statusLine }, details: ${ response.body }")
 
+  implicit class RichString(val s: String) extends AnyVal {
+    def toOneLiner: String = s.split("\n").map(_.trim).mkString(" ")
```

Comment:
I like this one! Maybe something to include in dans-scala-lib if it turns out we might use it more often (add TODO marker?).
On the other hand, if we use the convention that we send our json as a oneliner `String` anyway, we don't need this function. I know this is a more fundamental discussion, but IMHO it's something to consider...
@DANS-KNAW/easy opinions?